src/util/shared-prop-types.js

```diff
@@ -4,28 +4,42 @@ import PropTypes from 'prop-types';
 //       when they are imported and merged into a component's
 //       propTypes.
 
-export const childrenPropType = PropTypes.oneOfType([
-  PropTypes.arrayOf(PropTypes.node),
-  PropTypes.node
-]);
-
-export const childrenRenderPropType = PropTypes.func;
-
-export const stretchPropType = (props, propName, componentName) => {
-  if (props.inline && props.stretch) {
+export const eitherOrPropType = (
```

Comment:
LOL, we now have functions to create prop-type definitions?

Reply:
Yes 😎